lumicks/pylake/benchmark.py

```diff
@@ -112,8 +112,8 @@ class _Refinement(_Benchmark):
 
     def context(self):
         kymo_tracking = _generate_kymo_for_tracking(1, 4)
-        lines = lk.track_greedy(kymo_tracking, "red", line_width=1, pixel_threshold=20)
-        yield lambda: lk.refine_lines_gaussian(
+        lines = lk.track_greedy(kymo_tracking, "red", track_width=1, pixel_threshold=20)
+        yield lambda: lk.refine_tracks_gaussian(
```

Comment:
I'd suggest mentioning this in `changelog.md` since it's pretty noticeable on the user side when running `lk.benchmark()`.

Reply:
Good point. I'll add a line.